### PR TITLE
Stop build from leaving localpkg cache in src

### PR DESCRIPTION
**Pull request by wtgodbe**
https://github.com/dotnet/coreclr/issues/8130

### CODE REVIEW

File: dir.props [hunk 1]
```diff
@@ -67,6 +67,9 @@
     <PackagesBinDir>$(__PackagesBinDir)</PackagesBinDir>
     <PackagesBinDir Condition="'$(__PackagesBinDir)'==''">$(BinDir).nuget\</PackagesBinDir>
 
+    <PackageInstallPath>$(__PackageInstallPath)</PackageInstallPath>
```
Comment (weshaggard) [inline dir.props]: @karajas isn't there a way to disable the installing of this package instead of hacking the path here?

Comment (weshaggard) [inline dir.props]: https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets#L1217

Just set that property to disable installing this package locally as it isn't needed for you guys.

Comment (wtgodbe) [inline dir.props]: @weshaggard where's the right place to set that property?

Comment (weshaggard) [inline dir.props]: In the packaging.props would be the place I think to set it. It needs to be set in all the pkgprojs essentially.

Comment (karajas) [inline dir.props]: Yeah SkipInstallLocallyBuiltPackages for any packages that you don't want installed.

Comment (wtgodbe) [inline dir.props]: It looks like dir.props is the most reasonable place to set that property - the lowest level file that all the .pkgproj files import is the root-level dir.props (setting it in packages.builds did not work). I'll put up a PR to do so.

Comment (weshaggard) [inline dir.props]: OK. I guess you guys don't have a packages.props file like corefx. I would just put it where ever your other packaging properties are getting set.

**Comment by wtgodbe** [comment]
Updated

**Pull request merged**